samples/charts/tree-map/styling/App.razor

```diff
@@ -0,0 +1,41 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igComponent">
+
+        <Treemap Height="100%" Width="100%"
+                 DataSource="@Data"
+                 ParentIdMemberPath="Parent"
+                 IdMemberPath="Name"
+                 LabelMemberPath="Name"
+                 ValueMemberPath="Pop"
+                 LayoutType="TreemapLayoutType.Squarified"
+                 LayoutOrientation="TreemapOrientation.Vertical"
```

Comment:
remove layout props since they are covered in other sample

Reply:
pending